src/mmrelay/meshtastic/packet_routing.py

```
@@ -0,0 +1,288 @@
+                f"{setting_name}:empty_string",
+                "Ignoring empty string in meshtastic.packet_routing.%s.",
+                setting_name,
+            )
+            return frozenset()
+        return frozenset({stripped})
+    if isinstance(portnums, (list, tuple, set, frozenset)):
+        resolved: set[str] = set()
+        dropped_invalid_value = False
+        for entry in portnums:
+            if isinstance(entry, str):
+                entry = entry.strip()
```

Ruff (0.15.9)

[warning] `for` loop variable `entry` overwritten by assignment target

(PLW2901)